vagrant/elasticsearch-single.yml

```diff
@@ -46,6 +46,7 @@
     - "couchbase.username: couchbase"
     - "couchbase.password: couchbase"
     - "network.publish_host: {{ hostvars[inventory_hostname]['ansible_eth1']['ipv4']['address'] }}"
+    - “http.cors.enabled: true”
```

Comment:
@priteshpatel15 These looks like smart quotes and not regular quotes. This can happen if the file was edited in Word or copy/pasted from some WSYIWIG editors.